privx_api/privx_api.py

```diff
@@ -386,6 +415,27 @@ def create_user(self, user: dict) -> PrivXAPIResponse:
         response = self._http_post("userstore.users", body=user)
         return PrivXAPIResponse(response, 201)
 
+    def get_local_users(self) -> PrivXAPIResponse:
```

Comment:
This one is missing the parameters, please check docs.

https://apispecs.ssh.com/#/user/get_local_user_store_api_v1_users

Reply:
@kimmokssh : please check and let me know if all looks fine now.